app/src/main/AndroidManifest.xml

```diff
@@ -18,6 +18,8 @@
             </intent-filter>
         </activity>
         <activity android:name=".LoginScreenActivity" />
+        <activity android:name=".EmptyScreen"></activity>
```

Comment:
Since activity tag is empty, it's normal practice to close it with "/>" as opposed to using a closing tag. Please make changes as appropriate.